src/main/resources/assets/dreamcraft/lang/en_US.lang

```diff
@@ -1753,3 +1755,5 @@ dreamcraft.welcome.click_discord=Click to open discord!
 
 dreamcraft.pausemenu.bug=Report a Bug
 dreamcraft.pausemenu.wiki=Open the Wiki
+
+gt.blockcasingsNH.name=Air Filter Turbine Casing
```

Comment:
Could you leave this one off for now because of #1130 (and related PRs), I will include it in the GT5u lang file in that PR

Reply:
Sure